component/src/main/java/org/wso2/extension/siddhi/gpl/execution/streamingml/classification/hoeffdingtree/util/AdaptiveHoeffdingTreeModel.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (C) 2017 WSO2 Inc. (http://wso2.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.wso2.extension.siddhi.gpl.execution.streamingml.classification.hoeffdingtree.util;
+
+import com.yahoo.labs.samoa.instances.Attribute;
+import com.yahoo.labs.samoa.instances.DenseInstance;
+import com.yahoo.labs.samoa.instances.Instance;
+import com.yahoo.labs.samoa.instances.Instances;
+import com.yahoo.labs.samoa.instances.InstancesHeader;
+import moa.classifiers.trees.HoeffdingAdaptiveTree;
+import moa.core.FastVector;
+import moa.core.ObjectRepository;
+import moa.options.AbstractOptionHandler;
+import moa.streams.InstanceStream;
+import moa.tasks.TaskMonitor;
+import org.apache.log4j.Logger;
+import org.wso2.extension.siddhi.gpl.execution.streamingml.util.CoreUtils;
+import org.wso2.extension.siddhi.gpl.execution.streamingml.util.MathUtil;
+import org.wso2.siddhi.core.exception.SiddhiAppRuntimeException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Represents the Hoeffding Adaptive Tree Model
+ */
+public class AdaptiveHoeffdingTreeModel extends AbstractOptionHandler {
+    private static final long serialVersionUID = 1L;
+    private static final Logger logger = Logger.getLogger(AdaptiveHoeffdingTreeModel.class);
```

Comment:
Use Uppercase for constants

Reply:
Declaring its own serialVersionUID should follow the following format. Field name should be the same.
 ANY-ACCESS-MODIFIER static final long serialVersionUID = 42L;

ref, [docs/Serializable](https://docs.oracle.com/javase/8/docs/api/java/io/Serializable.html)

Also, with regard to all static final object reference types that are never followed by "." (dot). needs to be in UPPERCASE.

Although logger is a static final object reference as stated above, but because it is followed by "." everytime you use it, it can not be considered as a constant and thus should be lower case.